mscore/icons.cpp

```diff
@@ -182,8 +188,9 @@ void genIcons()
                   }
             }
 
+      // note that modifying the VOICES macro will not work here, but will fail at compile time
       static const char* vtext[VOICES] = { "1","2","3","4" };
-      int iw = preferences.getInt(PREF_UI_THEME_ICONHEIGHT) * 2 / 3; // 16;
+      int iw = preferences.getInt(PREF_UI_THEME_ICONHEIGHT) * 0.666666; // equivalent to * 2 / 3; // 16;
```

Comment:
except that before it was an integer operation

Reply:
Might be tempting to do such things, but if a PR claims to be about documentation on the tin, it should contain nothing but documentation ;-)

Reply:
ouch, you are right.

Reply:
amended